CHANGES.md

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ----------
 
+- Dune can now compose with installed Coq theories. Those theories need not
+  themselves be installed with Dune (#7047, @Alizter)
```

Comment:
Maybe we can add "Dune will now detect and use Coq theories installed in $coqlib/user-contrib, either by Dune or by other Coq build tools. Thus, composition with theories that are installed globally is now possible"

Reply:
What I mean is that we can present the change as:
1. we now detect theories outside the workspace
2. due to that, we can now compose with them too.